Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
Using `cmd` increases the my prompt latency by about an order of magnitude (1-2 ms to 30ms) when outside a Git repo.  Still that's not too bad.  That said, we have other git.exe invocations that could cause the same issue seems like we should think about clearing those errors.  Anyway I've updated this PR.  Play with it and see how it does. Heading to airport now.  :-)